cluster-sync/okd-4.3/provider.sh

```diff
@@ -6,17 +6,44 @@ function seed_images(){
   echo "seed_images is a noop for okd4.3"
 }
 
-ROOK_CEPH_VERSION=${ROOK_CEPH_VERSION:-v1.1.4}
-
 function configure_storage() {
+    _kubectl patch deployment -n openshift-machine-config-operator machine-config-operator -p '{"spec": {"replicas": 1}}'
+    _kubectl patch deployment -n openshift-machine-config-operator etcd-quorum-guard -p '{"spec": {"replicas": 1}}'
+  wait_machine_config_pool "True"
   #Make sure local is not default
+  set +e
   _kubectl patch storageclass local -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+  set -e
+  if [[ $KUBEVIRT_STORAGE == "ceph" ]] ; then
+    echo "Installing ceph storage"
+    configure_ceph
+  elif [[ $KUBEVIRT_STORAGE == "hpp" ]] ; then
```

Comment:
No multiple provisioner support?  Any situation in which we would want to have hpp/local for file and ceph for block?  Probably not.